dallinger/heroku/clock.py

```diff
@@ -28,9 +25,10 @@
 @scheduler.scheduled_job('interval', minutes=0.5)
 def check_db_for_missing_notifications():
     """Check the database for missing notifications."""
+    config = get_config()
```

Comment:
Yes, I didn't get a chance to run this on heroku yet, so there will be problems like this lurking.